examples/dataset/keypoint_detection/README.md

```diff
@@ -0,0 +1,43 @@
+# Example Integration: Keypoint Detection
+
+This example integration uses the [300 Faces In the Wild (300-W)](https://ibug.doc.ic.ac.uk/resources/300-W/) dataset
+to demonstrate testing keypoint detection models on Kolena.
+
+## Setup
+
+This project uses [Poetry](https://python-poetry.org/) for packaging and Python dependency management. To get started,
+install project dependencies from [`pyproject.toml`](./pyproject.toml) by running:
+
+```shell
+poetry update && poetry install
+```
+
+## Usage
+
+The data for this example integration lives in the publicly accessible S3 bucket `s3://kolena-public-datasets`.
+
+First, ensure that the `KOLENA_TOKEN` environment variable is populated in your environment. See our
+[initialization documentation](https://docs.kolena.io/installing-kolena/#initialization) for details.
+
+This project defines two scripts that perform the following operations:
+
+1. [`upload_dataset.py`](keypoint_detection/upload_dataset.py) creates the 300-W dataset on Kolena
+
+2. [`benchmark_model.py`](keypoint_detection/benchmark_model.py) tests a keypoint detection model on the 300-W dataset,
+  using either random keypoints (`random` option) or the open-source [RetinaFace](https://github.com/serengil/retinaface)
```

Comment:
Should probably mention somewhere in the readme that we are defaulting to random

Reply:
We're not — the model argument is required:

```
$ poetry run python3 keypoint_detection/benchmark_model.py
usage: benchmark_model.py [-h] {RetinaFace,random} [dataset]
benchmark_model.py: error: the following arguments are required: model
```